Repositories.md

```diff
@@ -0,0 +1,79 @@
+# Guidelines for Repository Inclusion
+
+This document outlines the requirements and guidelines for a Helm repository to
+be listed in the distributed search hosted by the Helm Project. This document
+may be updated at any time.
+
+## Process to Add A Repository
+
+There is a two step process for a repository to be added to the distributed
+search.
+
+1. Provide the repository's contact information to the Helm maintainers by filling
+out the form at ABC (link to be inserted). This contact information is necessary
+when there is a need for the Helm project to contact those who manage a repository
+(e.g., it goes offline). If the repository is owned by a team or company we suggest
+using a mailing list.
+2. Create a pull request against this repository adding the repository location
+in the file at ABC (link to be inserted).
+
+Once both of these are complete the repository and its contents will be listed
+in the distributed search.
+
+## Repository Best Practices
+
+Repositories listed in the distributed search should follow
+[Helm chart best practices](https://docs.helm.sh/chart_best_practices/).
+In addition to these practices we expect:
+
+* Every chart to have a maintainer listed along with valid contact information
+for each maintainer. The contact information could be in the form of an email
+address or link to a contact form
+* Charts to pass Helm lint, be installable and upgradable in all community
+supported versions of Kubernetes, and to use semantic versioning to share
+intent in released changes. To help with this we suggest leveraging the
+[Chart Testing](https://github.com/helm/chart-testing) project.
+* Have a NOTES.txt template with useful information which Helm will display
+after installation
+* Chart versions to be immutable
+
+## Legal Guidelines
+
+Helm is an incubating project under the Cloud Native Computing Foundation (CNCF)
+which itself is part of The Linux Foundation (TLF). TLF is a US non-profit operating
```

Comment:
s/TLF/The LF/

Reply:
@dankohn Can you give me a pointer on the docs on proper usage? The [trademark is for TLF](https://www.linuxfoundation.org/trademark-usage/) and the [privacy policy](https://www.linuxfoundation.org/privacy/) uses TLF. I was just trying to be consistent with usages in other similar places.